README.md

```diff
@@ -726,6 +726,9 @@ The first argument can be either a `url` or an `options` object. The only requir
 
 - `time` - If `true`, the request-response cycle (including all redirects) is timed at millisecond resolution, and the result provided on the response's `elapsedTime` property.
 
+---
+
+- `har` - A [HAR 1.2 Request Object](http://www.softwareishard.com/blog/har-12-spec/#request), will be processed from HAR format into options overwriting matching values *(see example below for details)*
```

Comment:
agreed, was going to do this first, then I followed the examples from `encoding` ...